draft-ietf-core-oscore-groupcomm.md

```diff
@@ -1294,19 +1407,33 @@ Upon receiving a request with the Group Flag set to 0, following the procedure i
 
 * If a new Recipient Context is created for this Recipient ID, new Pairwise Sender/Recipient Keys are also derived (see {{key-derivation-pairwise}}). The new Pairwise Sender/Recipient Keys are deleted if the Recipient Context is deleted as a result of the message not being successfully verified.
 
+* What is specified in {{ssec-verify-request}} of this document holds with respect to the following points.
```

Comment:
Is it needed to list these points here? The beginning paragraph of this section says _"The following differences also apply."_, so it is focusing on the differences. The points listed here seem to be the same as in {ssec-verify-request}

Reply:
Can be closed after discussions with Marco. (The diff is actually compared to Section 8.2 of RFC8613).